js/app/Spinner.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var $ = require( 'jquery' );
+
+var Spinner = function( $node ) {
+	this._$node = $node;
+};
+
+$.extend( Spinner.prototype, {
+	/**
+	 * @type {jQuery}
+	 */
+	_$node: null,
+
+	/**
+	 * Adds a spinner with `size` px height to a node.
+	 *
+	 * @param {int} size
```

Comment:
I am just curious but is this parameter actually required? it does not seem to be referenced to in the `add` functions body. And some calls of that functions pass some value of the parameter but i.e. when adding the spinner in the feedback form, there is no parameter.
I also removed those parameters where they are currently and it doesn't seem to affect the size of spinners (they look as good as before).